### ISSUE TITLE
[M1] Mutagen configuration for Magento 1

### ISSUE THREAD

**Issue by Den4ik** [davidalger/warden#97]
Currently just m2 mutagen configuration available.
Make sense to add configuration for other environment types.

**Comment by davidalger** [comment]
@Den4ik Just circling around to this. The 0.6.0 release will be including a new Shopware environment type with Mutagen configuration. The current images and setup will now support the flexibility needed to do this for more than just the Magento 2 environment type.

What types would you like to see Mutagen sync added for? I've [seen a request for it on Magento 1](https://github.com/davidalger/warden/issues/56) but that was scratched and deemed non-necessary. Open to adding this on other environment types, perhaps even just making it a standard for any type on macOS other than the `local` type where the user defines everything, just need to know the use-case, and for specific applications (like Shopware for example) what directories should be excluded from the sync (./var on M2 and Shopware is excluded due to being used for logs and cache files that would make the sync endlessly churn) and/or mounted via a delegated mount (pub/media on M2 is an example, or public/media on Shopware)

**Comment by Den4ik** [comment]
Hi @davidalger Big thanks for your work.
Only M1 required for me. I added it at my fork. Other types that I use at my projects currently available in warden

**Comment by davidalger** [comment]
Got it. Sounds like it may be useful to add a config for M1 then, and then call this one closed. This will leave Warden operating Mutagen syncs on macOS for M1, M2 and Sh6.

For M1, what I'll probably do (unless someone suggests differently) is similar to M2 and have:
* `media` mounted via delegated mount (and excluded from Mutagen sync)
* `var` excluded from Mutagen sync

What does the yml file your using for the sync on M1 look like currently?

**Comment by Den4ik** [comment]
```
---
sync:
  defaults:
    mode: two-way-resolved
    watch:
      pollingInterval: 10
    ignore:
      vcs: true
      paths:
        # System files
        - ".DS_Store"
        - "._*"

        # Vim files
        - "*~"
        - "*.sw[a-p]"

        # IDE files
        - "/.idea"

        # Magento files
        - "/media"
        - "/var/**"

    permissions:
      defaultFileMode: "0644"
      defaultDirectoryMode: "0755"
```

**Comment by davidalger** [comment]
@Den4ik This is implemented in the `develop-0.7` branch. Seems to work well in my testing on an M1 project I just on-boarded for a new client we're about to migrate to M2 over here. Let me know what you think. Planning to include this when I cut the 0.7.0 release, and didn't want it to block any potential bug fixes for 0.6.x in the meantime hence the branch it's on.
